### DISCUSSION TITLE
多个 Python 版本的加密脚本时报错

### DISCUSSION THREAD

**Discussion by 400dama** [dashingsoft/pyarmor#1445] [Q&A]
我使用的是windows10，pyarmor 8.3.4 pyarmor.cli.core 4.3.1
按照文档1.4.9对脚本进行多个版本的加密。分别在py37和py39下用python -m pyarmor.cli gen加密脚本成功。在使用python -m pyarmor.cli.merge -O dist dist1 dist2时报了如下错误：

> INFO     merging runtime files OK
> INFO     merging obfuscated scripts...
> INFO     handle "dist\startup.py"
> ERROR    unexpected character after line continuation character (<string>, line 1)

我尝试了在不同的版本下进行合并，都没有成功。请问我需要怎样解决这个问题呢？

**Comment by jondy** [comment]
使用调试选项之后的输出呢？`python -m pyarmor.cli.merge -d -O dist dist1 dist2`

**Reply by 400dama** [reply]
输出如下：
> INFO     start to merge 'dist37', 'dist39'...
> INFO     scan runtime package in the path "dist39"
> INFO     runtime package at "ABRuntime"
> INFO     merging runtime files...
> INFO     handle runtime package at "dist37"
> INFO     copy runtime files "py37" to "dist\ABRuntime"
> INFO     handle runtime package at "dist39"
> INFO     copy runtime files "py39" to "dist\ABRuntime"
> INFO     merging runtime files OK
> INFO     merging obfuscated scripts...
> INFO     handle "dist\startup.py"
> Traceback (most recent call last):
>   File "D:\tools\miniconda\envs\py39\lib\runpy.py", line 197, in _run_module_as_main
>     return _run_code(code, main_globals, None,
>   File "D:\tools\miniconda\envs\py39\lib\runpy.py", line 87, in _run_code
>     exec(code, run_globals)
>   File "D:\tools\miniconda\envs\py39\lib\site-packages\pyarmor\cli\merge.py", line 230, in <module>
>     main()
>   File "D:\tools\miniconda\envs\py39\lib\site-packages\pyarmor\cli\merge.py", line 219, in main
>     merge_paths(args.path, runtime_name, output)
>   File "D:\tools\miniconda\envs\py39\lib\site-packages\pyarmor\cli\merge.py", line 121, in merge_paths
>     merge_scripts(os.path.join(name, x), paths, dest)
>   File "D:\tools\miniconda\envs\py39\lib\site-packages\pyarmor\cli\merge.py", line 78, in merge_scripts
>     refcode = eval(refitem[-1])
>   File "<string>", line 1
>     \\"\xab\x90\x0b\xa0\xda\xb0\xe1\x8c\xb0\xe4\x84j\x94J.\x08PY005053\x00\x03\x07\x00B\r\r\n\x80\x00\x01\x00\x08\x00\x00\x00\x04\x00\x00\x00@\x00\x00\x00\x92\x10\x00\x00\x12\x89\n\x00W\x88\r\xef&6\xd0o\x1d\x0e5.j\x86H\xfc\x00\x00\x00\x00\x00\x00\x00\x00\xa7c\x06\x8e\xf4\xb1\xd5\xfa\x7fi\x8f\xfcN@\xf6\xd4c\x9by\xa8\xb8b\xaa\\W1C)\xe8\x08\xfa\xae\x83\x1c\x01\x87\xdb\x88\x01YG\x8c.\xb8\x07M\xeeA\xe8\xef\xbc\x9av\x1b\x86\xea\xf3S\xb3\xb9V\x8e\xb3\xfc\xcd\x94\x0e\xe0\x07\x7fL\xdf\xbaX\xb3\xb6\xeb\xbe\xbc\xa4\x07\xcb\x92\xae\xf6\xda\x99\x18\xbc\x08\xb870\x8a/\x02\x18yL\xed.\x9el\x02\x82\xc6,S\x0c\xc4N\x17\xd6\xbfwn\x1b\xf6H\xf0\xc0\x19\x80\x16\xfef9\xaf+}\xe7\x98\xfc\xe9\x0e\xf7\x98\x1a\x9f\xb9\x06\x9c\xe2!\x13\xe0\xc7l2_\x14\x00\x0ceg\xf2\xa2\x84&Q\xba\xa7J\xa7\xd7\xed1\x02\xd4b\xb0<\x84u\x97v\xb2\x93\x8a\x0c\xf2v\xe4\x81\x17\x07\xeb\x83OY_\xc1<\xb4\xfaS\x11\xb8\xe2\x01\x0en\x17\xcc\xc7\xdcR,B4\xfal\x05\xf1\xbe\x04\xc1\xdc\x0e\xc1\xec\x80\x8b,\x99&8S\xcb>\xd4\xb3D[\x08\x99\r\x07|\xbb"\x94\x81\xdc\x89\xb2\x9d\x9f\xee\xb8Y\xf7\xb9{Q\xa6\x93{h\x8d\x97X\xbb\x887_\xf3\xb7\xa6\x80\x9b \xb4^N\xf14\xe9Wjd\x93k\xa0\x8f"j\x97\xd1#m?\xf4\xbd\xab.?P\xee\x8e~\xf31ZB\x80y\xe4\x13\xb5Z\',\x8b\n\xd2\xc4\xfd\x0b\xcd@cwS\x01\xfaQn\x1bQ\x01\x18\xf0>J\x9a\xf7\xe1\x87\x9e\xd5\xb4\xd3\xc164\x8d\xab\xa2w\xc8\x8bp\xd2P\x9f>\x02\xf7[\x1e\xf6AOqo\xd6\xec\xc4\xfcD\xece\x84\x07\x05\xa1\xed\xb7\xf2A\xef\xab#\xe8b\xba\x1e\xe3VCI\xbf\xb4\xfb\xa0\x8eo>q\xc9\x1e\xad\xe6>3\xa6\xb0G.\x14\x1dZ\x02\n\xa6>\xbcR4\x9c\xc3\xb8w\xcf\x04\xc6\xce\xba\xc6n\xc8\xcf\xc2\x03\x1cJ\xa7\xdc\x0ez\xe3\xaf\xd0\xc2fZb,\x01\xbf\x11\xd9\xa3\xca\xbbU+\x9f\x98\x8aQ9|\x1b\x08\x91\xe6H\x0e\x8a\xfc\x0e\x96o\xef\xd0u\xa5\x0eR\xca^\x01\xc1\xd3\x12\x0e!o\x894\x92u\xca?w\n\xc4\x1d\xf9M\xd4\xc2\xe9\xff\xc0\x9f\xdej\xaeocP\xf3\xb8\x99\x87\r\t<\x13\xdc\xd5\xdek%H\xa4>\x80,\x8aT\x9a\x10x\xeb\x9c\xf1T\x84\xddt\xd4X\x9e\x01;\xc2hz\xbc\x05\xf9\xd4\x06\xa0T\xfb\xda\xb8\xe5\xf0\x89w\x8c\x0e\'\xca\x06\xa8\x1b\x950\x8f\xdd\xf9\xaf\r=\xcb\xc7(\xf0{<V\xf1\xff\x18\xcb\xbb\xa2{z\xfd\x1d+\x9c\x1f\x98\x14\xfe\x13U\x1f|\xc6\xb5\xba\xee\xb2\xb4\tG\x88\xb2\x04\x1c|\xd6<v \xcaf\xf1\x8b\x90\xcfF\x03Im\xd3\xbevf\x84\xff\xc4\x9f\xa4#Nj\x89\x03\x1c\'+\x91\xc4\x90\xa8\x06H"o\xd7]\x05\x88\xb9\x8fH\xecB\x87\xae?\x9dKI]\xd6N\xe4@\x14$\x92\xf7\xc4\xbf\x06\xd5\xd1\xf4\xe9. \xfc\x1d\\{\x05\xa3\xc4\xec\xebt\x8c\xc7\x9e7\xbc.\xbdx\xfe\x84\x11s\xe9\xfd\x9c\xad\xec4\xa9\xe7\xb7\xe6\x8f\xf7\xcb\xc4\xce`:\xb70\xd1qS\xddG\xeek%\xdb\xe9\xfb\x87_\xc8Y\x8eD\xa3rE\xa1\xa5\x1a\xd7\x81\x01\xe7\x08\x87\xf7\xb3\x0f\x00\x8f\x80\xfc\xab\xf9\xe6/oT\x91\xc0\xda\x0b\x0e\x9eb0\xbe\xeaI\xd7\xe8[\xbe\x84\x18\x80\x81\xd2bB\xc0,\x10\xc7~\xd9\xab\xe6\xf6<44\xda!\x8f\xe4\x1b\x95\x11\xcd.\xc5\x9a\xc7:\xfd\xd33\xec\xba\xcc\xac\x9a\x057H\x1a;nR{\xb6\xfcr\xb1\xd9Nt\xdb\xd0\x91\xab\xa7\xf5\x88\x8e\xf8\x8c\xd5%\xa8=A\xa3\x82\'\x7f\x06\xdeK\xa5>\xa2\x9c\x14nS\xe9\xad\xe2~\xfd\\\x12*m\xfcu\x7f\x9e\x82e\xcf\xcd\x95\xf3h\xa4\xae\xfbM=\xda\x90\xdb\x10\x82P\xb6>\xac\xd1\x1d\xfeo\x84\xf2\x14Gs\xab=p\x17?\xfd\x9a/\x85b\x86\xb5\xce\xd24\x1f&\xa6z\'\x9d\x9f\x8d{\xb8\xbd\x8f_%5R\x10\xff\xd7T\xae^\xba\xba\r\xcc\xfd v\xe30\x8b\t\xdc\x1c\x80\xc2\x1b\xab\xf3"\x98\x05\xea\xec\rIL\xdd\x83B\x057\x94G\x93\xd2\x81\x02\xce\xdcWd\xb4\xec\x9b\xb7o&\xa3\xc2h^\x99\xc22\xdc\x95P\xf3\x8d(H\x188\xc0\x15V\x9exK\xfd\xbao<\xe6e\xa4\xce\xbbY\x8b\x8bC\x9e*2\xa1\xb8\x9a\x12\x11L\xfa5M\x90o\xc0D\xc6\xdfqN\xc0\xb3jsy\x01\x91\xbf&}#\x97\x9e"\xd5x\xf7\xcb\x96\x07J\xc4\xad\\\x81\xed\xa0\xf3\xb4-oaF8\xad\x9b\xd4\x8d\xfc\x90|\x15c\x93\xd8\x92\\,\x80\x96A\x99=?\x01\x96\xc4\x0e\x16\x91\x15\x84C\x98\x08\xa1!\x08\xf2\xbc\xc2\xe8\x826\xa5\xc1\x15\x07\x1d&^\xa2[f\x14#\xf7\x9a\xf9\xb9\xe0\xc9@\\\x9e\xab\xab\x11C\xee\x84I\x10v\xaa\x7f\xa1\x18\xee\xe6-~\xe5\xe5\xc1\xc7\xd6:Qf4.\xcc\xa0\xce\x07A\x19#Z8o\x8d\xe9^9\xcc&\xfc\xda\xbe\x15oE\xa7\x8dt\xf3o\x83\xea\x8e{\xd9~\xd5X\xa7\x1b\xbc]\xcc\x83/dph\xde\xf1\xa5N8N\x86\xf8`&\xbd\x9ax\x9f\xb9\xb5\xf3\xfc\xe9\x8cR!,\x14\xb7\x92\xd3\xb6b\xcb\xb5\x196\xf8\xa6X\x0e\xeb\xec\x12S\xd0#|\xd7\xed>X0\x83W\xa0\xbb \xad\xb1\x84\x9aJ2E\xf5\xf2\xf2\xfc\x84\xb7\xfb\x0c\xa9^\x9b\xed\xed\xef\xd5\x99\xd0UK\xd4\xe0\xfe\xf4\x96\xe49E\xa0$B\x1cv\xfc=\x85\xb6\xc0\x13\x0f\x08\xf9\xbb\xa1\x7f$G\x85\x15\xc3\x19N\xa6\xbbD\x07\x81ZZ\xd7m\xe7M\xf7?\xe5\x1c\xe4`~\xd8\x9c\x85\xads U\xce\xca\x17\xfb\xde\r\xf9O\x9eNuWQ\xb9\x8a\x99\xd1.1\xb8\xda\xee\xf7M\x13T\x7f\xf1\xeeH\x03\xcc\x93%"\xbd\x15F_\xf3}\xa8h5/\xe7Y\x14\xf5B\xf5GyU\xac\xba\x07\xa6Q|P\x99L\x9cz\xac\x19\xfb\xdaS\x91\xd9_6M\x92\x11\xa4\xed\xec\xf9\xeb#\x01"]\x12\xf4\xe4\xbfg\xc9q\xb7\xe0\x82\xa8\xff\x17p\xff\x0b\xa16\x8b\x7fN\xf0\xc0\x02]\x92+\xa7\xdb\xf2p\xaa^\xf4\xd3i\xf2\xfa\x8bzR;sC\xe61\x1a\x1a_\x9b\xf3\xcfP\xf380\xe9*\xfcuL\xfc~\xe6i\x1f\x05\xcc\x16\xdc\x96\x95\xecS\'\x98\xd5\xfb\xd9 <\xce\xcd\xd6\xe6\xe2\x16g\xf0\xef]G\xe4iV\xf5\xe9Y\xeac\xe0\xda\x13XG\x13\x8a\x9d,2\xcb\r\xbd#S\xa7\xd5\xb8\xfaD\xd3\xfc\x1c\x87\xf9\xaa\x1f\x0b!\x96\xec7\xc2\xa1\xfb\xf6\x1bJ&\xcfn\xf5F\xf5G\x0e5T\x82Q\xcf\xec\xa7\x99}\x0e\x01qP6\xdf\xe2\\\xff\x1d?M\x13\x8bpS\xec^\xb3\xfc/<z\xd0N\n\xc2B\xd4\x15\xa5\xed\xb8\xb4\x0b\xa3\xb7\xc8\xdc\x95AR\xd6k~\x10*\x0eTMx\xef\x07\xfa\xda\xa3\xf4\xe8\x9duq!\xc6\xa9\xd8\x1f\x81d\xf5\xf2)\x8c\xdb\xff\xeaD\x83\xba\xc8\xba\xc7T\x9b\xfc\xd8\xd3l\xcf\xd9\x82\xda\x19\xad\xb7\xa7p\x03x,o\x97\xeb\xc3\xc2#\xc8p\t\x93;e~6\xeb\x85\x95^~o\xf0BUh\xfd\nC\x7f\x16csJ\\0\xc7D\xc2\xceQd\xe3z\xba\xeb\xfd\\\xed\xe1\xfd\x17\x81?\x07\x03[\x86\xad<D\x15\xe4\xe2\xcd68t5\x1e\x96\x82f+I\n5\xff?\\\xea.^\xa8#\xfc\x9a\xcb\xd0\xa8\x03b#\xb8 \xe1\xf4\xe3u\xd2\xc3\x17\xc3\\xO\xf8>\xb3\\\xfe\xe9i\x88\x1fY\x83\x8c\x88\xa4\xb2\xbb7\xff`\x0b\xb5\x80e\xbb\xaf\x0e,\r)\x88\xbb\x92:\xa8#PQ!g\x96\xc5\xc2\xf4\xf5\xf0<s\xeb\xb6\xa0\xa0\x84\xd8{\xd4\xffy\x03\xeb\xc2\t\xd0[\xbb\xb9\x83\xcf\x80J\x82K#U\xdc\xc9\xb5\x19~\x05[\x15\xc9q\tY=\x13\x85\x91\xe5c+M\xc1\xcf\xe4\x1c&\xdf[\x8b\x8a\x0f\xae\x12\x9e\x16\xa5~,x\xa1\x83Zu@\xc6\x9d\xf6\xb6\xcf\x86\x1a,H\x8eD\x89\xc8j\xc2*@r\x8em\xf8\xe7\x9b@*h\xe9@\xdd\xc8\xd3\x15\xfd\xff\xa8b\xc6d\x15\xf0\x9f\x17b6\x18\xeaJ\x1c\xb4le\xe9\x87\x96SA\x1a\x16\xcf\xec\xa6\xce\xeb\xd5X\xb8$\xca\xe4f\x0fm&\xfbz\xdc\xa6\xda\x8f\xfb\xb0\x1a!\x01\x1atU\x18\x00b\x9fL\xa0d\xd3\x18\x7f\xa3\x18\xf0\xd7.\xac\x9e\r\xbd\x14\xc8\xe9&m\xdf\x92\xa1\x88\xaaH7\x16{\x9b\xe1\x1fL5\x946L{_\x12\xfdg\xc4\xdel\xfd\xb2)U$\x9a\xd5b\xb6\xb1\xed\x7f\x89\xae\x02\xa5w\xe8\xc4\xfe\x05\xf6\xc2\xd7\x02\xf5 \x98\xb7\xddk\xa0\xb4\xaf\xd3\n\x80\xe1\xec\xbc9\xcdUz\x86\x02\xa7\x8e@\x12\x95P\x91\xf3q\xaa\x9d\xfbOW\xe4.\xae?F\x04\xb5\xff\xa4\xf3c\xe2\xe7\xcd\xf1i)\xc6\x19\x81\xfcj:}v\xd1-\xcfD\x05\x00,\xd1"\xf1/\x8e\xcf\xf8\x11Z\x0b\xa4\xf4\xc7Vac\\\x85P\xea\x13\xf1C8\x9e"\xd2\x05E\xce\xff\xfb\xff\x9f\x12\xdc\x83\xf4\xd9\xb7g53\xde\xa9\xb3u^\xf6l\xaf\xd1}8\x1b\x1b\xfae\xac\xea\xb9\x10\x00%2\xfe8\xe9`\xa0HV\xa7\x1f7\xa4\xec\xa9R\xfe\x04[F\xfegM\xae]9.\xab\x0fK\xd7\xf7\x18|\xc7\xc2C$\xd2\x7fTA\xc4\xcb\x88\xfd\xc0\x01\xfe$\x9cS\xe3\xab\xbf\xb1\xc0\x9a9\xd8\xf8\x1b\xddE\x15\xbd\xae\xda*mV\x03\xd9X\x12\xe5\x11\x01F\xe2-\x8d\xaf\xb1\x12L\x89\x8a\']\x9a\x96\xcd\x96\x03\xb2\xba\xe2"\xd7\xb6l\x15\x8c\x9a\xd2\x955\x144\xf0\xa0\n\xa4k\nK>hZm\\P~}\r\xc5\xb9\\\xdd&\xe0\x9a\xb4\xf0G\xb0\xe8\xaf\xeaR\x87I/\x84\x12:\'\xdf\xfe\xe0\xdd\x83\xef:\x14\xbb.\x98[U*\xf9\x1a\x0e\x87\xc1\xc1\xe6\xe2B\xdcp\xcaA\x01)i\r\xd7l8\x08\xb0:\xcc?NS.\x96\xd5\xeb\x91\xd1\x83PJ\xe3\x8c\x01x\x16\xebm\xa5L\xbf\xe0\x05\xb1i\xf7\xbe\xd6\xe1\xa0\x91\xf2\xf6\xd4\x993$IQ2Y\x9c?\x97\x87\x94\xdf\xc4\x95?\xde\xab\x8bOk\xa1\x8e\x8bD\xef}\xd3Z\xca\xe6\x8e\x8a\xe0\xf2S\x05y\x8b\xbbF\xc3gT\xc0V:i\x83U(L7N\x0f\x98\xd3r\xe5`\x8c\xfdi\xef\xa3xi\xd6.\xd3K\xb2l\xf3Cs\xae\xa7\x87\x95\xe1\xe3f6\xbb\x0f\x1b\xabd}\x98e \xc5\x10\xc5\x85Y\xa5O\x7f\xc7\x14\xcc\x06`\xb8\x92\xae\x8f4\x94\xaf{\xa4T\x84W\x0b\xbc\xc8>\xa4\xa6\xb8\'\xa1\xaf\xe9pI\xb9\xd7\xb0~\xacB\xbc\xa3\x85\xc0HM\x07j"P\xa6\xd6\xe8\xe1\xd6\x1e\x9f\xc2\x838\xbb\xee\x8a\xee.\x02nw\x0eKN*8-\x1f\xb4Z\x105\xce\xe7\xe2<\xde\xdd\xe4|V\xe5\xc0\x1e\xc1\xc4\xf6\xa2\n\x11|f\x0f\xc8\x11\x94<\xbcP\x00\x017\x02:\xb3r \xe8L\xec\x08E\x18\xf5\x83\x92\xf0\x0c*\xf5\xc3c \xe4\xf0L\xcb\x8fm|\xfd\xa8R\xdal\xbb\xc75L\xd1\xb1\xaf"9"R\xff\xf5\x91E\x15]\x97\x1b\xaaj^\\\xd0\xc2+\xb4\xe2V\x15a\xe3\x13\x90\xc6\xdf\xadcXtp\xb7\xeb\xc9!\xaa\xaf\xd1\x8d/\xec\xcc7\xe1`3\x9epj\x108tR\r\xa7\x13\xbc\xbe\x1cf\xb6\x94\xbfs\xcb;\xaf\x0eI?s\xa0d1\xc2.\xe7%15\xe5\x91\x89\xf8<\x15\xf0\xd4\xf6q\xbb}\xe2\xa8k\x88\xfd<\xb8\'tS\xc9\x17"\x80~\xcd\xdd\x8c\xa3V\xcf\xdfU\x9d\xb3\xb1dN\xde\xdeD\x94"\xde\xa7.-\x88c\x00\x15\xa4\x01\xfd\x7f\r\x94D\x03\xb9E2S\x1f\xaf\xb1\x82\xd9\xd8\x12S\xc4\xd9.xjZ#\xa9i77\x0c\xe9\xef\xa5v\x00\x1e\xd9\xb4sz\xb0Zi\xf8\'\xc8!\xf6\xcf\xb5\x8f\xf7\xd5]8\x8dc\x8d7\xf4o\xae$\x06\xbei?\xbe\xba\xa9\xfb@m9n<\xb63\xc7\xa4\xb3N\'4*\x0c\x1by\xb0L\xfc\x19\n\x8e\x10\xe2\x148\x0e\xa1\xd9\x18\x07z\x04\xc4\x08\xa5\x87\x06\xd6|\xeb{\xefa[]\x97~\xda6h@_=\x8bT\xa0\x1av\x90\r\xa6\x94S\xd8\xc2z\x8c\xd6\xcd\xd7\x8a\xa4*%\xb7\n\xcaQ\xd1\xd4\x11\xb1K\xa6\xf58ym\xd2\xff\xc5Y`\x07\xf7)\xbc\x8d1\xf6\xe2x\x97\xbbO\xb6>RC\xb6I\xed;\xb40}\xb0W=\xdc\x7f\xef\x1b,yG\x97\xaa")\x00\xb6\xa2\xb2\xf0\xa6%\x8c{d\xff=\xbay\x06=\xf5\xe8Y\xc4\xe0\xbe\x7f\xfc\xfe\x9dtb|8Y\xeeO\xa5\x15SQf\xb9HO+\r\x17 \xfcA\x00|\xa5\xab\xc24\x82\xac\x91\x0e\x18$\x1e\x83.\xca\xf5Q\xa6F\xc5\x15\xfa\xe9\x06\xbdoc+\n\x01\x9d\x86NKS\x1a$I\x15*"z\x8b\xff\n\xb6V\xb4\x15t\xa7\xddt\xb8\xe1s*\xd3\xd2\xeeO\x82Y:\xcf\xe6\xf5\x02\x16\xc0\xdd\xac\x89]\xf2fb1\xd1\xf7K\xe5\x95"\xba\x12\xf9C\x8f\xa7G\xcb$X\xe4:\xe6\xb59,\x90L\xd9\xfb\x0b\xa0M5Q\xb6fc\x1d8\x05$}\xde\xb2\xc6\xccK\x03`\x9a\xd9\xb9\xec_r\x14\x95 \xc5\x82\x8f%\xe6\xc7\xd8|/G\xd9\xc0\x8c\x0f\x84\xa4\xbc\xf2!\xc7S\xd88\x88\x13\xe8\xa8\xf0u5;\x86\x04HO\xc2\xb0y^\x1bM{\xfd\x9ao\xdf\xe7\xbe\x11>A=\xbc\x95\xf4.\x1as\xd4\xc05k|\xd0\x1f\xa0\x05\x1e\x12\x1dd\xca\xa0\xb89\xf4#c\xf2\t"\x197\x19_\xb2Ii\x02\x0f2\x14\x90\x1d\xee\x9d\xc8\xdf\x9ckM\x87\xebu\x93dr%\xb2\xbb\xcd\xac\x00\x84\xfb\xae\x88\xe0\x91\xcd\xf0,\x1f\xe0\xfa7\xee\xf5\x93\x14\xce\xe3\xd1\xa9\xd5N\x93\xde\xbb\xdfu:!\xe7K\xbb\x10F\xc0ck\x90W\xeaG\x85B=\x15\x1fD\xcd+\xec\xe5\xd9\x9f\xbeA\xcc\xc8\xcc\xa6\x9d\xd3\x02\x88\xfc/:1\x14\xd2\x0e1\xef\x91X\xca*\x13\x8adf\x16\xc6\xbfA\x91g\xb0\xd7\x14\xf8\x1fJ\x01\xa4\x9cL\x1fb\xd6\x00m\xf1<\x9b5GCWn\x82\xd9R\n\xdd\xday\xcf\xd7\xd5\x1e\xedLQ\xde\xb6o|\xca\x8c\xdfY\xfev\xb3L\xf6ty\xb6B\x7f\xb6q\t\x1a\xc2\xa7fw\xec+\xcf\x95\x97w#W\\\xeb-\r}%\xad\xad\x04\xd7h\xcdH1\xd9{\xddQ\x06^YZ\x14!\xdf\x15|\xbd \xf4\xbc\xcdG\x9aN|(?\x11\xe6\x7f\xe7\x8d\xe1\xc9\xc6\x98\x1a\xe0^\xaa\xdf\x92\xaf\xf5\xf3\x17F\xb9~)\xed\xe2\xffT\xf3Q\xb0\xba\x13\xf1\x98\xf5eDN\xb8\xb7\xe5\x83\x9f\x96\x95N\xabp\x16\xa4\x96eSX-\xbcD\xe5\x1f\xa77\x025\xabI\x95\'\xf94\xc4\xde\'\x1b\x19\x80\xaa\x86\xec\xa0Cw\x1e\x0e\x17\xf4!I6~\x9f\x8f\x87z\xb96lQ\xfe\x87V!\xda(P\x198\x10\xc6\t||\x1c\xe7\x05}\xd8\xdb\xe3\xf2\xf7\xff\x99\x19h\tx\xd5\xa6a\xf6\xae&\xf8\xb3\xaf\xcfM\x16O\x98\x01j\xdc\xfeJ\xc8\x93\xda\xf9H\xc5\x87\xebsFFE\xb2\xb2\x9f{"Ln\x96\xa5k\xcf\xf7(\xf2\x864\x0e\x97\x83\x9b\n\xdc\xa9\xb0\xfa\xf0\x19\x80\x8br\x98\x80/$\xdf\xb1\x10\xfc\x1c\x05\x93b]]\xf86\xd5\xa2".\x01}\xfe\xdd\xedMQ\xf1rx\x89\xc3E\xa0Q\xcc\x92\x0f6*\x96\xf3zW\xb9\xa97\xa5v)\xb5\xfd\x11\x01\xb4(\xd7w\xfe$\xbd\x1c\xf5\xd1\xc9g\x81\\Nr\x8e{\x88F\xda7\x9a\xa4\x98\r>\x8cK\xbb\x00\x90=\xdf\xb2\xf0m\xad\xbd\xd0\xdb-\xdfs\xe9}i\x8f\xf2\x8e\x971\x81\xbd\xa7\xbe\xf2\xa89\xf5\xbb\xc9\x99\xd0^\x8b\x16\xa5>\xfbw\x9bg:n\xf4\x13\xbe\x06^k\x15\xf0\x88w\xdc\xb5\x11\x9c<w\xd2\x8e\x8f!\xe7\xeb!\xbbf\xc92\xd8LI;\xd0w\x0f\x0bC\x93R\xe8\tL\x94\xe3\xdd*\xf9t\rQ\xb8\xc8\x98)\x0b\xb4\xe9#\xa9\x93\x0c\xbfM\xb7*\xb8v\xbf\xc8t\xfc\xa8?\xa1\xa9\xa5\xce^\'r\x85\xe2l\xee[\xe4\x858`\xf0\x8bA\x18\x1dE\x99\x17u+\xf1\xbeY]\x83=\xce\xf4\xe8\x18~U\xc5\xd1\x8c\xf2o\x8f\x81\x9dS\x19\x94\x08\x1b!\xde\x18\xf7D\x85[\x99\xf8\x87M\xcf\xcf\xb8\xc6XC9\x03o2\x01\x16\xd8R\x92\x0eM`}Ok\x9cT\x08\x88\x9c\x04\\\xb7\x9f\xe6^i\xb8\xd0\xb4\xb5>\x01\x0c\xa9\xd0.\xfe\x0b\xd0O\xad\x13\x9d\xce\x0cW\xec\xa8Wb\xeb\x83\xd2\xa3=\xb2\xdf\xcdk\xba\x9e\xe8\xb8\x96\x12\xb2\xe8\x7fy\xcd)n\x9ar\x1b\xcd3\xd5k\xc3l\xd3\xc1m\xb1Ch\xd7\xfc\xbb\x90\xc2\xb7\xf5W\xa0\xef\xa78\xc7\x17\x05\xbbbb\xbd\x00Yv^\xdf\x7fdB\rF\xb7\xfd\xbd\x07\xb4\x90,\x1f,\x892\xbf\xe9\xacm\x1b\x0f\xbf\n\xeb0<p\xfdl\\\x92\x18W\t\xe3\xed\x0e\xad\xb9\x11\xe7\x8f\xd9,-\x11\x99Tw\xf3f|\xc9p\xff\xde\xa6\x8d\x8fe\x0fa#\xe1\xeco\xa5\x81\xa3\xfe\xf3?\x9a\xf6\x93\xab\x0b|\x86\xd3\xa7\x88^\xb6O\x8ej\x91\x94\x9d\x0e:M\xae\xed\x86\xcb\'\xe9\xd1+\xdc\xd2)\xad(\xb4\x8d7\x1c~?\x0f\xf4\xa5\xdb \x19\xd7 G\xefgs\xa3\xb1\x82\xea\xcf\xc4\xbc\xf5\xddU\xde\xe9\xe7H#\x7f\x99\x18+x\xd1\xd8O\x1a\x05V\xfb\xe8o\x81+x\x86\x12\xce\xb0\xcf]|'
>      ^
> SyntaxError: unexpected character after line continuation character

**Comment by jondy** [comment]
装一下开发版本看一下，修改了一下 merge.py
```
pip install https://pyarmor.dashingsoft.com/downloads/temp/pyarmor-8.3.5.zip
```

**Reply by 400dama** [reply]
用8.3.5重新merge了一下，这次倒是没有报错了。 不过在 import 脚本的时候有了下面的报错

> import startup
> 
> Error: Traceback (most recent call last):
> Error:   File "\<string>", line 1, in \<module>
> Error:   File "D:\AB\dist\startup.py", line 2, in \<module>
> Error:     from ABRuntime import \_\_pyarmor__
> Error:   File "D:\AB\dist\ABRuntime\\_\_init__.py", line 3, in \<module>
> Error:     \_\_pyarmor__ = \_\_import__("py%d%d.pyarmor_runtime" % py_version[:2], globals(), locals(), ["\_\_pyarmor__"], 1).\_\_pyarmor__
> Error: SystemError: D:\_w\1\s\Objects\listobject.c:213: bad argument to internal function

**Comment by jondy** [comment]
首先确保合并之前的脚本可以正常运行

**Reply by 400dama** [reply]
我发现原来是开了BCC模式后就会报上面那个错，但关了就没事了。所以多Python版本加密是本来就不好开BCC了吗？
